setup/setup_native.sh

```diff
@@ -5,10 +5,7 @@ source setup/setup_shared.sh
 
 export PLATFORMS=""
 
-if [ $SETUP_IOS ] || ( [ !$SETUP_ANDROID ] && [ !$SETUP_IOS ] ); then
-    OSX_MAJOR_VERSION=`sw_vers | grep ProductVersion | cut -d ':' -f 2 | cut -d '.' -f 1`
-    echo "Found OSX major version" $OSX_MAJOR_VERSION
-
+if [ $SETUP_IOS ]; then
```

Comment:
Doesn't this just go back to the behavior we had before, where ruby (and by extension, cocoapods) are only installed on iOS builds? This just happens to work on GitHub actions because the default image has ruby installed. But it will not work on a brand new laptop, for example.